### PR TITLE
Removing sensitive info from logs, and adding explanatory messages

### PR DESCRIPTION
**Pull request by Lukehub1**

### CODE REVIEW

File: src/main/java/uk/gov/laa/pfla/auth/service/services/ReportTrackingTableService.java [hunk 1]
```diff
@@ -46,11 +46,8 @@ public synchronized void updateReportTrackingTable(int requestedId, LocalDateTim
                 .reportGeneratedBy(user.getGivenName() + " " + user.getSurname())
                 .build();
 
-        if(log.isDebugEnabled()){
-            log.debug("reportTrackingTableModel to string: {} ", reportTrackingTableModel);
-        }
-
-        //Create a trackingtable row in the table
+        //Create a new trackingtable row
+        log.info("Updating reportTrackingTable with details of the report request from user");
```
Comment (deepujames1234) [inline src/main/java/uk/gov/laa/pfla/auth/service/services/ReportTrackingTableService.java:50]: From a security point of view Is it required to mention table name in the info logs? can we remove table name? mention something like "updating report tracking details" in you want to keep this in info log

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportViewsDao.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ public class ReportViewsDao {
     @NotNull
     public List<Map<String, Object>> callDataBase(String sqlQuery) throws ReportIdNotFoundException {
         List<Map<String, Object>> resultList;
-        log.debug("Calling database for result list, with sqlQuery: {} ", sqlQuery);
+        log.info("Calling database for result list, with sqlQuery: {} ", sqlQuery);
```
Comment (deepujames1234) [inline src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportViewsDao.java:27]: Is it required info ? Think from security point of view in prod logs

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java [hunk 3]
```diff
@@ -25,19 +25,20 @@ public ReportTrackingTableDao(JdbcTemplate writeJdbcTemplate){
...
     public void updateTrackingTable(ReportTrackingTableModel trackingModel) {
 
+
         JdbcTemplate localJdbcTemplate = this.writeJdbcTemplate;
 
         String sql = "INSERT INTO GPFD.REPORT_TRACKING (ID, REPORT_NAME, REPORT_URL, CREATION_TIME, MAPPING_ID, REPORT_GENERATED_BY) VALUES (GPFD_TRACKING_TABLE_SEQUENCE.NEXTVAL,?,?,?,?,?)";
         Timestamp timestamp = Timestamp.valueOf(trackingModel.getCreationTime());
 
         //Insert values into sql statement and update
+        log.info("Updating tracking table");
```
Comment (deepujames1234) [inline src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java:35]: ### From a security point of view Is it required to mention table name in the info logs?

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java [hunk 4]
```diff
@@ -51,15 +52,14 @@ public List<MappingTableModel> fetchReportList() throws DatabaseReadException {
             throw new DatabaseReadException("No results returned from mapping table");
         }
 
-        log.debug("Result list, a list of objects each representing a row in the mapping table: {}", resultList);
```
Comment (deepujames1234) [inline src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:54]: From a security point of view Is it required to mention table name in the info logs?

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java [hunk 5]
```diff
@@ -42,6 +42,7 @@ public List<MappingTableModel> fetchReportList() throws DatabaseReadException {
 
 
         try {
+            log.info("Reading from mapping table");
```
[the remark this answers is not in the document]
Comment (Lukehub1) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:45]: the name of the table is CSV_TO_SQL_MAPPING table, so we havent logged the table name?

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java (hunk 4 above)
Comment (Lukehub1) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:54]: the name of the table is CSV_TO_SQL_MAPPING table, so we havent logged the table name?

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (Lukehub1) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:45]: Could log 'getting mapping data' maybe?

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java (hunk 3 above)
Comment (Lukehub1) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java:35]: The name of the table is REPORT_TRACKING table, so we havent logged the table name? If i log  'updating **report** tracking information' like you suggest, thats arguably giving more information isnt it?

How about just 'updating tracking information' ?

Comment (deepujames1234) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/ReportTrackingTableDao.java:35]: Looks good

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java (hunk 4 above)
Comment (deepujames1234) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:54]: Looks good then

File: src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java (hunk 5 above)
[the remark this answers is not in the document]
Comment (deepujames1234) [reply src/main/java/uk/gov/laa/pfla/auth/service/dao/MappingTableDao.java:45]: Looks good

**Review by deepujames1234 — APPROVED** [review]

**Pull request merged**